### ISSUE TITLE
Adding further option values to 856$x

### ISSUE THREAD

**Issue by HirschSt** [rism-ch/muscat#669]
To clearly seperate digitalization links from other types by adding to the existing "IIIf" something like:

- [ ] Digitalizisation of musical notation
- [ ] Other

**Comment by HirschSt** [comment]
Proposal: 
-------------------------------------------
Englisch terms:
Digitized music (without IIIF)
Digitized music (with IIIF) 
Other 
  
german:
Notendigitalisat (ohne IIIF)
Notendigitalisat (mit IIIF)
Sonstiges

856 $x soll bitte ein Pflichtfeld sein, wenn 856 $u ausgefüllt ist.

**Comment by jenniferward** [comment]
As long as it's clear that "Digitalization" is just a code and not the proper term (because that describes the process, not the digital object itself).

**Comment by xhero** [comment]
I made a small script that looks into the IIIF links and verify they are actually IIIF. If it is OK I would go on and make a script that sets the non-IIIF ones to nothing, so then we can apply one of Stephan's proposed codes. What do you think?

**Comment by xhero** [comment]
Apparently we only have **_three_** (!) links that are real IIIF json data...

```
100126
300044017
400014686
```

**Comment by HirschSt** [comment]
@xhero To my knowledge at least all uris from A-SPD should be IIIf, eg https://muscat.rism.info/admin/sources/1001001628. Can you check this?

**Comment by lpugin** [comment]
Not quite. The link points to their Mirador viewer that itself loads IIIF images. The link to the manifest is actually https://iiifpresentation.minerva.arz.oeaw.ac.at/musikarchivspitz/A-SPD_A15/manifest

It would not be bad to have both links, one as IIIF and one as digitized resource.

**Comment by xhero** [comment]
We can have both pretty easily :) The validation could run in the background? Or I could also try to access the resource from ajax on the fly, but that could probably create a bit of lag for the user.

**Comment by HirschSt** [comment]
Having both sounds good, @xhero changing by maintenance? IMO we have no alternative to the ajax call since external resources are blocked by Berlin infrastructure... maybe dealing with all possible errors are tricky too...

**Comment by xhero** [comment]
yes we should have a small maintenance task to fix the wrong records.
You are right about ajax, I can test it to see if is too slow

**Comment by lpugin** [comment]
@HirschSt back to the issue thread, sorry for starting a discussion in the commit. I think having "manifest" explicitly is better because we want to make sure users understand it is a link to a json manifest, and not to a resource (viewer) using IIIF.

**Comment by jenniferward** [comment]
@lpugin @HirschSt  Please send me a detailed explanation - I cannot derive one from the comments above -  and I will add it to the guidelines.

**Comment by lpugin** [comment]
Exactly. See https://muscat.rism.info/catalog/100126 for the images shown in Muscat with Diva.js

**Comment by jenniferward** [comment]
@lpugin what about your comment "It would not be bad to have both links, one as IIIF and one as digitized resource": should this be in the guidelines?

**Comment by lpugin** [comment]
Yes, having both a link to an external vewier and a link to the IIIF manifest is good. But I would rephrase it. "For cases where both links to an external viewer and to a IIIF manifest is available, both links should be given separately."
